tedge/src/cli/connect/common_mosquitto_config.rs

```diff
@@ -40,6 +40,11 @@ impl CommonMosquittoConfig {
 
         Ok(())
     }
+
+    pub fn with_port(self, listener: u16) -> Self {
```

Comment:
rename `listener` to `port`, otherwise it's a bit confusing IMHO.

Reply:
done